pynicotine/gtkgui/ui/popovers/searchfilterhelp.ui

```diff
@@ -394,7 +393,8 @@
                     <property name="visible">True</property>
                     <property name="xalign">0</property>
                     <property name="margin-start">12</property>
-                    <property name="label" translatable="yes">To exclude non-audio files use !0 in the duration filter.</property>
+                    <property name="label" translatable="yes">Non-audio files can be excluded:
```

Comment:
`Non-audio files can be excluded` not relevant anymore, we have generic file type filters.